tests/pfcwd_test.py

```diff
@@ -3,6 +3,8 @@
 import sys
 
 from click.testing import CliRunner
+from unittest.mock import patch
```

Comment:
`unittest` is part of the standard library. Please move this line to the bottom of the first group of imports, e.g., 

```python
import imp
import os
import sys
from unittest.mock import patch
```